Inconsistent error responses in tests

```
--- FAIL: Test_root_RunE/options (0.00s)
        root_test.go:234: STDERR: To regenerate the on-disk snapshot, re-run the tests with `UPDATE_TESTY=true` or `-update-testy` or `-tags=update`.
            --- expected
            +++ actual
            @@ -3,3 +3,3 @@
             CouchDB options: %vmap[bar:baz foo:bar]
             [ping] Will ping server: "http://localhost:5984/"
            -Error: Head "http://localhost:5984/_up": dial tcp [::1]:5984: connect: connection refused
            +Error: Head "http://localhost:5984/_up": dial tcp 127.0.0.1:5984: connect: connection refused
```

--- a/cmd/kivik/cmd/root_test.go
+++ b/cmd/kivik/cmd/root_test.go
@@ -258,6 +258,10 @@ var standardReplacements = []testy.Replacement{
 		Regexp:      regexp.MustCompile(`go\d\.\d+\.[\da-z-]+`),
 		Replacement: `goX.XX.X`,
 	},
+	{
+		Regexp:      regexp.MustCompile(`dial tcp 127\.0\.0\.1:`),
+		Replacement: "dial tcp [::1]:",
+	},
 }
 
 func (tt *cmdTest) Test(t *testing.T, re ...testy.Replacement) {

